### ISSUE TITLE
network name and description

### ISSUE THREAD

**Issue by WayneCrawford** [FDSN/StationXML#159]
On the FDSN.org site, both a "Network name" and "Description" are specified for a given FDSN code.  But, in StationXML, only "description" is allowed.  More confusingly, the StationXML "description" often corresponds to the "Network name" and not the "Description" (see example screenshots below, taken from the fdsn.org website and the https://www.fdsn.org/webservices/fdsnws-station-1.1.pdf example.

I think that either a "name" field should be added to StationXML <Network>, or the fdsn.org site should rename "Network name" to "Description" (and "Decription" to "Long Description"?)

![FDSN_IU_website](https://github.com/user-attachments/assets/8472d460-f415-49e5-b02b-f7a7adf4f19d)

![FDSN_IU_XMLexample](https://github.com/user-attachments/assets/73fc6f7a-84ec-4e4c-be52-2081541e211f)

**Comment by crotwell** [comment]
Seems reasonable. 

Currently Station has a description, but also a name inside <Site>.  Perhaps adding <Site> to Network would also be useful, where the "site" in this case means the whole geographical area for the network? That would allow the use of other tags like <country> that may be useful for national networks.

In the meantime, perhaps put the name into description and the longer description into a <comment> with subject being set to "long description" or something similar. 

Alternatively, setting website in the network operator at least allows people to find more description if desired.

**Comment by megies** [comment]
Everybody please keep in mind that changes to the standard have a lot of downstream implications (e.g. to software tools supporting said standard) that can force tough decisions like causing backwards incompatible changes that affect thousands of users writing code.
On the other hand text fields on websites that are designed for humans reading them and not necessarily for automated applications can easily shuffle around field names without disruptions.

Adding new fields/tags in the standard isn't the worst for downstream applications but making changes to existing fields, their meaning and semantics creates problems for maintainers and users.
I guess what I'm trying to say is, we should try and keep standard changes to a bare minimum whenever they can't be avoided but in cases like this I'd massively argue it's better to adjust the webpage to use the existing standard first of all.

**Comment by crotwell** [comment]
Yes, but I suspect the chances of a stationxml revision in the next decade are really small. So this is more about remembering things that might need to be updated if/when a revision happens. The FDSN rarely moves very fast! :)

Use of `<Comment>` seems the best for holding longer descriptions for foreseeable future.
